doc/thebigpicture.mm

```diff
@@ -125,9 +125,6 @@
 </node>
 </node>
 <node COLOR="#000000" CREATED="0" ID="ID_1157254381" MODIFIED="1533054088445" POSITION="left" TEXT="API">
-<node COLOR="#000000" CREATED="0" MODIFIED="0" TEXT="gocean0.1">
```

Comment:
Possibly we should just get rid of this altogether - we don't use it any more. What does you think @rupertford?

Reply:
As discussed, in this PR I will just get rid of this particular node, but we agree this can be removed in a separate PR.